packages/augur-state/src/Controller.ts

```diff
@@ -1,5 +1,6 @@
 import { DB } from './db/DB';
 import { Augur } from 'augur-api';
+const uploadBlockNumbers = require('augur-artifacts/upload-block-numbers.json');
```

Comment:
What are upload block numbers? I'm worried that this will make testing difficult unless it's added to the Controller class via its constructor.

Reply:
These are the block numbers on which the Augur contracts were uploaded. We start scanning the blockchain for event logs at this block number because there cannot be any that exist before that block.

Reply:
Ah makes sense. I see that there are network IDs that start at 0, 1, and some other low values so this won't affect testing 👍.